src/common/components/notification-modal/NotificationModal.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from "react";
+import classNames from "classnames/bind";
+import IC_NOTIFICATION from "@/images/ic_notification.svg";
+import GetNotificationData from "@/shared/hooks/getNotificationData";
+import styles from "./NotificationModal.module.scss";
+
+import AlertModal from "./notificationModal-components/AlertCardModal";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationModal() {
+  const { iconColor, alertCount } = GetNotificationData();
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleToggleModal = () => {
+    setIsOpen(!isOpen);
```

Comment:
이전 상태값을 기반으로 상태를 변경할때는 이렇게 표현해주시면 좋습니다~
```suggestion
    setIsOpen(isOpen => !isOpen);
```